hubmap/frontend/cypress/integration/hubmap_spec.js

```diff
@@ -32,4 +32,32 @@ describe('HuBMAP', () => {
     // Footer:
     cy.contains('Supported by the NIH Common Fund');
   });
+
+  describe('The Login Page', function () {
+    before(() => {
+      cy.server();
+      const api = 'http://localhost:8000/';
+      cy.route(`${api}/api/?format=json`, 'fixture:base.json');
+      cy.route(`${api}/api/colors/?format=json`, 'fixture:colors.json');
+      cy.route(`${api}/api/genes/?format=json`, 'fixture:genes.json');
+      cy.route(`${api}/auth/login/globus`, 'fixture:login.json');
+      cy.setCookie('email', 'test@gmail.com')
+    });
+
+    it('sets auth cookie when logging in via form submission', function () {
+      // click login
+      cy.get('login').click()
+  
+      // we should be redirected to / (home)
+      cy.url().should('include', '/')
```

Comment:
I think `include` would be a substring anywhere in the URL: Use equality, like https://github.com/hubmapconsortium/hubmap-data-portal/pull/96/files#diff-f65b9ef05f60b40bb52f0b693f2c225fR41